### ISSUE TITLE
Antlr4-runtime with no GUI support

### ISSUE THREAD

**Issue by badbob** [antlr/antlr4#1160]
Hi, everyone. 

I developing some Android application and make simple grammar for search criteria with SQL-translating with Antlr4. (Thanks all contributors - great tool) But, I can't include _antlr4-runtime_ artifact into Android application build (.apk), because it has _java.awt_ package dependencies. And there is no such packages in Android SDK. 

Is there any build of _antlr4-runtime_ artifact with no GUI features at all?  
Or some walkaround?

**Comment by badbob** [comment]
The issue fixed by using 4.5.3 version of antlr4-runtime. Previously I used 4.5 which caused error: 

* Fix Multiple dex files define Lorg/antlr/runtime/IntStream
* antlr4-runtime-4.5.jar: Invalid package reference in library; not included in Android: java.awt.event. Referenced from org.antlr.v4.runtime.misc.Utils.2.
* antlr4-runtime-4.5.jar: Invalid package reference in library; not included in Android: java.awt.geom. Referenced from org.abego.treelayout.TreeLayout.
* antlr4-runtime-4.5.jar: Invalid package reference in library; not included in Android: java.awt.image. Referenced from org.antlr.v4.runtime.misc.GraphicsSupport.
* antlr4-runtime-4.5.jar: Invalid package reference in library; not included in Android: java.awt.print. Referenced from org.antlr.v4.runtime.misc.GraphicsSupport.1.
* antlr4-runtime-4.5.jar: Invalid package reference in library; not included in Android: java.awt. Referenced from org.antlr.v4.runtime.misc.GraphicsSupport.

**Comment by kaeptmblaubaer1000** [comment]
@badbob please reopen,
```
  The first 3 errors (out of 13) were:
  /home/travis/.gradle/caches/modules-2/files-2.1/org.antlr/ST4/4.0.8/a1c55e974f8a94d78e2348fa6ff63f4fa1fae64/ST4-4.0.8.jar: Error: Invalid package reference in library; not included in Android: javax.swing.text. Referenced from org.stringtemplate.v4.gui.STViz. [InvalidPackage]
  /home/travis/.gradle/caches/modules-2/files-2.1/org.antlr/antlr4/4.7.1/51ef964b44323720bef371d569796af1f4e225ac/antlr4-4.7.1.jar: Error: Invalid package reference in library; not included in Android: java.awt.event. Referenced from org.antlr.v4.gui.TreeViewer.1. [InvalidPackage]
  /home/travis/.gradle/caches/modules-2/files-2.1/org.antlr/antlr4/4.7.1/51ef964b44323720bef371d569796af1f4e225ac/antlr4-4.7.1.jar: Error: Invalid package reference in library; not included in Android: java.awt.image. Referenced from org.antlr.v4.gui.GraphicsSupport. [InvalidPackage]
```
